react/components/admin/pages/Form/Form.tsx

```diff
@@ -148,13 +156,20 @@ const Form: React.FunctionComponent<Props> = ({
             disabled={!isInfoEditable}
             label={intl.formatMessage(messages.seoDescription)}
             onChange={detailChangeHandlerGetter('metaTagDescription')}
+            resize="vertical"
             value={data.metaTagDescription}
           />
           <FormFieldSeparator />
           <Select
             creatable
             disabled={!isInfoEditable}
+            formatCreateLabel={(keyword: string) =>
+              intl.formatMessage(messages.createKeywordsMessage, { keyword })
+            }
             label={intl.formatMessage(messages.seoKeywords)}
+            noOptionsMessage={() =>
+              intl.formatMessage(messages.noOptionsKeywordsMessage)
+            }
```

Comment:
Does this prop really expect a function? `noOptionsMessage` sounds like a `string` prop.

Reply:
Yep: [Select props](https://react-select.com/props#select-props)